applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -199,8 +199,13 @@ config SBC_CHANNEL_MODE_MONO
 		Only mono is supported at the moment
 
 endmenu # SBC
+
+config SW_CODEC_PLC_DISABLED
```

Comment:
I would move this to line:147 instead, better to have all the options related to sw_codec together

Reply:
Done!